cypress/e2e/SignInPassword.cy.ts

```diff
@@ -28,4 +29,28 @@ describe('Email and Password Validation', () => {
 
     cy.url().should('contain', redirectionLink);
   });
+
+  it('Sign In With Password shows success message if no redirect', () => {
+    cy.intercept(
+      {
+        pathname: API_ROUTES.SIGN_IN_WITH_PASSWORD_ROUTE,
+      },
+      (req) => {
+        req.reply({ statusCode: 303 });
+      },
+    ).as('signInWithPassword');
+
+    const { WRONG_EMAIL, WRONG_PASSWORD, GRAASP } = MEMBERS;
+    cy.visit(SIGN_IN_PATH);
+    // Select sign in method
+    cy.signInPasswordMethodAndCheck();
```

Comment:
Without the comments I'm not sure to understand the difference between `cy.signInPasswordMethodAndCheck` and `cy.signInPasswordAndCheck`. What about `cy.selectSignInWithPasswordAndCheck` (maybe a bit to long?) instead of `cy.signInPasswordMethodAndCheck`?

Reply:
I removed this method, because there is no more switch button. The two layouts are displayed directly on the same interface.